packages/react-components/package.json

```diff
@@ -16,17 +16,20 @@
   "scripts": {
     "build": "just-scripts build",
     "bundle": "just-scripts bundle",
+    "chromatic": "chromatic --project-token $CHROMATIC_PROJECT_TOKEN --exit-zero-on-changes",
     "clean": "just-scripts clean",
     "code-style": "just-scripts code-style",
     "just": "just-scripts",
     "lint": "just-scripts lint",
-    "start": "just-scripts dev:storybook"
+    "start": "just-scripts dev:storybook",
+    "build-storybook": "just-scripts storybook:build"
   },
   "devDependencies": {
     "@fluentui/eslint-plugin": "^1.0.1",
     "@fluentui/scripts": "^1.0.0",
     "@types/react": "16.9.42",
     "@types/react-dom": "16.9.10",
+    "chromatic": "5.6.3",
```

Comment:
I'd prefer not to add this as a dependency which must be installed for everyone every time, when it's actually only used for releasing the docs. Could the chromatic comand be changed to something like `npx chromatic@5.6.3 <args here>`?